StreetCare/StreetCare/Views/Screens/OutreachFormView.swift

```diff
@@ -0,0 +1,312 @@
+                        
+                        Text(NSLocalizedString("maximumCapacity", comment: ""))
+                            .font(.headline)
+                        
+                        TextField(NSLocalizedString("egmaximumCapacity", comment: ""), text: $maxCapacity)
+                            .textFieldStyle(RoundedBorderTextFieldStyle())
+                            .keyboardType(.numberPad)
+                        
+                        // Event Description with character counter
+                        Text(NSLocalizedString("eventDescription", comment: ""))
+                            .font(.headline)
+                
+                        TextEditor(text: $eventDescription)
+                            .frame(height: 200)  // Larger frame for description
+                            .border(Color.gray, width: 0.5)
+                            .cornerRadius(5)
+                            .onChange(of: eventDescription) { newValue in
+                                if newValue.count > descriptionLimit {
+                                    eventDescription = String(newValue.prefix(descriptionLimit))
+                                }
+                            }
+                            .padding(.bottom, 5)
+
+                        HStack {
+                            Spacer()
+                            Text("\(eventDescription.count)/\(descriptionLimit)")
+                                .font(.caption)
+                                .foregroundColor(.gray)
+                        }
+                        .padding(.bottom, 5)
+                        
+                        NavLinkButton(title: NSLocalizedString("selectSkillsButtonTitle", comment: ""), width: 300, secondaryButton: false)
+                            .padding()
+                            .background(Color.clear)
+                            .onTapGesture {
+                                isSkillSheetPresented.toggle()
+                            }
+                        
+                        if !selectedSkills.isEmpty {
+                            Text("Selected Skills:")
+                                .font(.headline)
+                            ForEach(selectedSkills, id: \.self) { skill in
+                                Text(skill)
+                                    .padding(.horizontal)
+                                    .background(Color.gray.opacity(0.2))
+                                    .cornerRadius(5)
+                            }
+                        }
+                        
+                        NavLinkButton(title: NSLocalizedString("saveEventButtonTitle", comment: ""), width: 300, secondaryButton: false)
+                            .padding()
+                            .background(Color.clear)
+                            .disabled(!allFieldsFilled)
+                            .onTapGesture {
+                                saveToFirestore()
```

Comment:
When saving successfully or when discarding can you dismiss the previous view and return to the Community screen


Uploading Screen_recording_20241119_172927.mp4…